.travis.yml

```diff
@@ -1,3 +1,4 @@
+sudo: false
```

Comment:
Ahora dice: This job is running on container-based infrastructure, which does not allow use of 'sudo', setuid and setguid executables.